explainer.md

```diff
@@ -546,11 +546,11 @@ It’s important to realize that all of the alternative solutions offer no metho
 
-In the meantime the group that developed WebVR recognized the issues with the initial API, in part through feedback from developers and standards bodies, and worked towards resolving them. Eventually they recognized that in order to create a more scalable and more ergonomic API they would have to break backwards compatibility with WebVR. This new revision of the API was referred to as WebVR 2.0 for a while, but eventually was officially renamed **WebXR** in recognition of the fact that the new API would support both VR and AR content. Developement of WebXR has been able to benefit not only from the group's experience with WebVR but also from a more mature landscape of immersive computing devices that now includes multiple commercial headsets, the emergence of both mobile and headset AR, several competing and mature native APIs, and the work of other standards bodies addressing the same space.
+In the meantime the group that developed WebVR recognized the issues with the initial API, in part through feedback from developers and standards bodies, and worked towards resolving them. Eventually they recognized that in order to create a more scalable and more ergonomic API they would have to break backwards compatibility with WebVR. This new revision of the API was referred to as WebVR 2.0 for a while, but eventually was officially renamed **WebXR** in recognition of the fact that the new API would support both VR and AR content. Developement of WebXR has been able to benefit not only from the group's experience with WebVR but also from a more mature landscape of immersive computing devices that now includes multiple commercial headsets, the emergence of both mobile and headset AR, and multiple mature native APIs.
 
-The intent is that WebXR will completely replace WebVR in the coming years, with all browsers that initially shipped WebVR commiting to shipping WebXR in it's place once the API design is finished. In addition, the WebXR Polyfill provides an implementation of WebXR that can utilize WebVR if the browser expoese it instead, allowing developers to make use of the newer API in all cases.
+WebXR is intended to completely replace WebVR in the coming years. All browsers that initially shipped WebVR have committed to shipping WebXR in it's place once the API design is finished. In addition, the WebXR Polyfill provides an implementation of WebXR that can utilize WebVR if the browser exposes it instead, allowing developers to make use of the newer API in all cases.
```

Comment:
One more nit...
>In addition, the WebXR Polyfill provides an implementation of WebXR that can utilize WebVR if the browser exposes it instead

In the meanwhile, developers can code against WebXR, relying on the [WebXR Polyfill](link url) to ensure their code runs in browsers with only WebVR implementations.

Reply:
Doh! I meant to make this a link originally. Thanks for catching that, and for the better wording.